### PR TITLE
Add Badge components

### PR DESCRIPTION
**Pull request by yash9263**

### CODE REVIEW

File: styles/components/badges.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  height: 16px;
+  border-radius: 50%;
+  border: 1px solid var(--color-neutral-100);
+  background-color: var(--color-danger-400);
+}
+
+.i-badge {
+  display: block;
+  text-align: center;
+  min-width: 28px;
```
Comment (Hamza-777) [inline styles/components/badges.css:16]: Try to use rem as much as possible. Or any responsive unit.

**Review by Hamza-777 — APPROVED** [review]
The code looks fine. Good work.

**Review by vrx29 — APPROVED** [review]
Looks good you can merge

File: styles/components/badges.css (hunk 1 above)
Comment (yash9263) [reply styles/components/badges.css:16]: Thanks for pointing it out.

**Pull request merged**